packages/block-editor/src/components/block-selection-clearer/index.js

```diff
@@ -9,7 +9,7 @@ import { useRefEffect } from '@wordpress/compose';
  */
 import { store as blockEditorStore } from '../../store';
 
-export function useBlockSelectionClearer() {
+export function useBlockSelectionClearer( onlySelfClicks = false ) {
```

Comment:
Why this change was needed here?

Reply:
Without it, clicking a button inside the frame would deselect the blocks and lose focus (like clicking an "appender" button) (A number of e2e tests were failing, that's how I discovered it)